osv/vulnerability.proto

```diff
@@ -152,13 +152,16 @@ message Credit {
 message Reference {
   enum Type {
     NONE = 0;
-    WEB = 1;
     ADVISORY = 2;
-    REPORT = 3;
-    FIX = 4;
-    PACKAGE = 5;
     ARTICLE = 6;
```

Comment:
Also ordered these lexicographically

Reply:
Please add a comment with what the next ID is, given it's now much more difficult to determine this.

Reply:
done!